canonical-web-and-design/juju.is
# Add dynamic tutorials page

## Done
Added the dynamic tutorials docs and removed the hardcoded one.

## QA
- Go to the tutorials section in the demo
- Check it lists all tutorials tagged juju
- Click on a tutorial
- Check it all works

Fixes https://github.com/canonical-web-and-design/juju.is/issues/49
